jml/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM centos:centos7
+
+RUN yum -y update
+RUN yum -y install \
+  awscli \
```

Comment:
Are you using awscli/libgomp?

Reply:
This is now obsolate, as we've removed docker.  (but yeah, Z3 uses libgomp)